src/libraries/System.Runtime/tests/System/DateTimeOffsetTests.cs

```diff
@@ -1157,7 +1202,7 @@ public static TestTime FromSeconds(DateTimeOffset dateTimeOffset, long unixTimeS
         public static void Ctor_Calendar_TimeSpan()
         {
             var dateTimeOffset = new DateTimeOffset(1, 1, 1, 0, 0, 0, 0, new GregorianCalendar(),TimeSpan.Zero);
-            VerifyDateTimeOffset(dateTimeOffset, 1, 1, 1, 0, 0, 0, 0, TimeSpan.Zero);
```

Comment:
Could you please add extra test here which call the new constructor taking microseconds and calendar too?